package-lock.json

```diff
@@ -708,9 +708,10 @@
       }
     },
     "@little-universe/do-not-allow-missing-properties": {
-      "version": "git+ssh://git@github.com/little-universe/do-not-allow-missing-properties.git#fefee6a3c812250b8110f2be68b3622ddb45c132",
-      "from": "@little-universe/do-not-allow-missing-properties@github:little-universe/do-not-allow-missing-properties",
+      "version": "github:little-universe/do-not-allow-missing-properties#c693139a4cfef435100b2229bb6cd2688fd7847f",
+      "from": "github:little-universe/do-not-allow-missing-properties",
```

Comment:
Hopefully this solves this on my side:

![image](https://user-images.githubusercontent.com/10826197/173438764-ca243980-b906-4f4f-8898-3f75cd12127b.png)